src/main/java/fact/features/source/PixelSetForSourcePosition.java

```diff
@@ -0,0 +1,83 @@
+package fact.features.source;
+
+import fact.Constants;
+import fact.TriggerEmulation.SumUpPatches;
+import fact.Utils;
+import fact.container.PixelSet;
+import fact.coordinates.CameraCoordinate;
+import fact.hexmap.CameraPixel;
+import fact.hexmap.FactPixelMapping;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import stream.Data;
+import stream.Processor;
+import stream.annotations.Parameter;
+
+
+/**
+ * generate a pixel set from the coordinates of stars in the camera given by the starPositionKeys
+ * Created by jbuss on 30.08.18.
+ */
+public class PixelSetForSourcePosition implements Processor {
```

Comment:
could you guide me to the place where you use the `StarService` because in the cleaning the `StarPositionsKey` is still used:
https://github.com/fact-project/fact-tools/blob/7c675d6ea21ec9b2f1f0e854984b571a3c02a1dc/src/main/java/fact/cleaning/TwoLevelTimeNeighbor.java#L60-L61